### PR TITLE
Data browser - Various styling changes and additions

### LINKED ISSUE #912: Add further styling changes to the console.
TODO:
- [x] Tab link style - make it clear which one is selected and which ones are available to click
- [x] Using standard controls - checkbox for diff, not slider; dropdown for asc/desc
- [x] Third row valid time is ill-aligned
- [x] Drop full timezone in VT/TT history
- [x] Drop 'back' button
- [x] Ctrl-Enter to run query (including tip)
- [x] Entity/query back to tabs
- [x] 'no results' should be smaller so that you don't need to scroll to see the bottom
- [x] Removing dropshadow/rounded corner styling on boxes
--> Currently 1px solid black, can change as we see fit.
- [x] Add Crux header
- [x] Move history to bottom right of textarea
- [x] vt/tt in query editor as rows to make it clear that the date/time fields are related
- [x] Hide VT/TT by default (icons bottom right of textarea)
- [x] Green submit button
- [x] Lato font family

### PR DESCRIPTION
**Pull request by danmason**
Resolves #912

### CODE REVIEW

File: crux-http-server/src-cljs/crux/ui/codemirror.cljs [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                        :autoRefresh true
+                        :value @value-atom
+                        :theme "eclipse"}]
+          (reset! cm-inst (js/CodeMirror. el opts))))
+      :reagent-render
+      (fn []
+        [:div.CodeMirror-readOnly
+         [:div.textarea
+          {:class class}]])})))
+
```
Comment (luciodale) [inline crux-http-server/src-cljs/crux/ui/codemirror.cljs:121]: This can go :)

File: crux-http-server/src-cljs/crux/ui/events.cljs [hunk 2]
```diff
@@ -64,19 +98,44 @@
                         {:valid-time (common/date-time->datetime vtd vtt)
                          :transaction-time (common/date-time->datetime ttd ttt)})
                        (remove #(nil? (second %)))
-                       (into {}))]
-     {:dispatch [:navigate :query {} query-params]})))
+                       (into {}))
+         limit (:limit query-params 100)
+         current-storage (or (reader/read-string (.getItem js/window.localStorage "query")) [])
```
Comment (luciodale) [inline crux-http-server/src-cljs/crux/ui/events.cljs:103]: Maybe we should do this (103) in a re-frame interceptor

File: crux-http-server/src-cljs/crux/ui/codemirror.cljs (hunk 1 above)
Comment (danmason) [reply crux-http-server/src-cljs/crux/ui/codemirror.cljs:121]: We are using it, though (in the query history view)?

**Review by jarohen — CHANGES_REQUESTED** [review]
Thanks gents :smile: 

As usual, I've only skimmed the CLJS/CSS side, assume you two have paired up on that? If not, happy to review more thoroughly.

Any chance we can serve the Lato font from the Google fonts CDN?

**Comment by danmason** [comment]
Post our discussion/attempts at doing so, we decided on serving the fonts from the repo. Generally gone through and cleaned up the unused fonts, bringing the total number of font files added down to 16 rather than round 70.

**Review by luciodale — APPROVED** [review]

**Pull request merged**